media/css/firefox/profile-per-install.scss

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import "../../../node_modules/@mozilla-protocol/core/protocol/css/includes/lib";
+@import "../../../node_modules/@mozilla-protocol/core/protocol/css/components/article";
+@import "../protocol/components/fxa";
+
+.mzp-c-article {
+    margin-bottom: $layout-lg;
+
+    &.mzp-l-content {
```

Comment:
Whilst this is largely leaning on existing article component styles with a couple of small overrides, it's not something we have clearly defined in Protocol yet. We should probably make sure we fix https://github.com/mozilla/protocol/issues/232 and then follow up here to apply these new styles? It's a small thing, but my concern is the more of these paper cuts we create then if we forget about them, we'll end up with a bigger wound than we anticipate?

Reply:
Yup, it's a problem. We keep getting designs that are almost-but-not-quite Protocol. The ideal solution is to make such design changes in Protocol first. That's how a design system is supposed to work. But we're not given the time to follow the best procedure so we hack and rack up technical debt.

This page should have been in production last week and we really can't wait any longer. So I can either go against the intended design and use only what's already in Protocol, or we can keep the tweaks to ship it as designed and take on the added debt.

Honestly I'm not sure which is best. If we de-tweak this page and change things Protocol later we'd probably still have to revisit the page to implement the changes, even if it's just adding a new layout class. So in that case maybe we should just keep the tweaks and take the debt.

Amy suggested we should keep a running list of all the Protocol pages in bedrock, and we can include what components are in use on them, with notes about any tweaks that were made to each one. Keeping such a list would have the dual benefits of 1) making sure we don't forget about pages that need to be updated when Protocol changes and 2) building a strong case for how much technical debt we're incurring when we need to ask for time to work on Protocol.

Reply:
Yeah at this point I think we just have to run with the custom changes. For future work, I think we need to push back harder on custom designs if they want us to rush work in. These are both really basic pages visually, so probably could have been built with existing components.